### ISSUE TITLE
BDMV playback broken with Leia

### ISSUE THREAD

**Issue by teotikalki** [xbmc/xbmc#16811]
### Describe the bug
Here is a clear and concise description of what the problem is:

I just upgraded to Kodi 18.3. BDMV folders in my library no longer play as they used to and I cannot find how to recover the 'good' behavior.
'Play' used to play the movie. I could stop the movie and resume playback from the stopped point. Last night I was inspired to rewatch the epic fight scene from Ip Man (the first BDMV I've played since the upgrade).
I can no longer hit play and actually get to the movie. I am forced to watch several logos and a long intro scene before the bluray menu appears. Once in the movie, hitting stop returns to the Kodi menu but trying to resume from where I left off simply plays again from the beginning. This disc seems worse than Ip Man 2, where hitting the new 'video menu' button actually goes to the video menu.
This is especially frustrating as I originally started using Kodi explicitly because I needed something to handle BDMVs.

With a little bit of further testing, the 'Video Menu' button seems buggy. It often freezes playback, takes 10+s to appear, and many times clicking the button just seems to do nothing.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

I recall a context menu entry that used to be available that gave options to either 'play main movie' or 'play bluray menu'. As a collector with many discs with extra features I have actually wanted the bluray menus to work for YEARS... but most of the time I just want to watch a movie! The option to set as default the old behavior which has worked well for years seems like it should be a given (play main movie). The option to launch the disc menu instead should exist in the context menu. Resuming playback should actually RESUME playback, not restart it.

## Actual Behavior

When I play the movie (eg Ip Man) the player shows an 8s video, a 14s video, etc (logos) that cannot be bypassed or fast-forwarded. When I eventually get to the movie and stop, Kodi gives me the option to resume but it replays the movie from the beginning (of the INTRO garbage, not the actual movie)

## Possible Fix

As stated above, the most obvious fix is to have back the behavior of Kodi 17 where it would simply play the movie. There should be an option to launch the bluray menu that will allow for things like disc extras to be viewed. Selecting which is my default behavior should be possible.
...I'm fairly certain I remember these possibilities from before? When bluray menus didn't actually work?

### To Reproduce

1. Upgraded to Kodi 18
2. Played movie stored as BDMV that worked before
3.

### Debuglog

The debuglog can be found here:
Really? This is a behavior change....

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [X] Linux
 - [ ] OSX
 - [ ] Raspberry-Pi
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Debian 10
 - Kodi version: 18.4

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by DaVukovic** [comment]
We will need the logfile

**Comment by teotikalki** [comment]
@jjd-uk Thank you, that's exactly what I was looking for! I hadn't looked at the 'discs' menu because I wasn't actually playing from the disc (sort of thought that section was like 'PVR' - used for hardware I didn't connect to kodi. I'm pretty sure that the three choices from that screen used to be available in the context menu? I seem to recall that and have never changed anything in 'discs' before.
I don't see how the setting could have been changed, though. I am the only user. The last time I watched a movie before upgrading from Krypton was when I got Captain Marvel last week, and the behavior then was the same as it has been for the last few years on this system: 'play main movie' (bluray menus didn't work properly before though). The only thing I changed was upgrading to Leia.
Anyways, thanks again. :)

@DaVukovic We really didn't.
